plugin-build/build.gradle.kts

```diff
@@ -70,3 +70,34 @@ if (gradle.gradleVersion >= "6.6.0") {
     val publish = extensions.getByType(MavenPublishPluginExtension::class.java)
     publish.releaseSigningEnabled = BuildUtils.shouldSignArtifacts()
 }
+
+subprojects {
+    apply<DistributionPlugin>()
+
+    val sep = File.separator
+
+    configure<DistributionContainer> {
+        this.getByName("main").contents {
+            from("build${sep}libs")
+            from("build${sep}publications${sep}maven")
+            from("build${sep}publications${sep}sentryPluginPluginMarkerMaven")
+        }
+    }
+
+    tasks.named("distZip").configure {
+        this.dependsOn("publishToMavenLocal")
```

Comment:
need to validate if this is the right name, `./gradlew tasks` would show the right names

Reply:
just did, its right